config/routes.js

```diff
@@ -126,6 +126,8 @@ module.exports.routes = {
     // API
     'get /api/topics/:topicId/examples': 'ExampleController.findByTopic',
     'post /api/topics/:topicId/examples': 'ExampleController.postExampleToTopic',
+    'post /api/:exampleId/remove': 'ExampleController.deleteExample',
+    'post /api/:exampleId/update': 'ExampleController.updateExample',
```

Comment:
So the route looks like this: `/api/1/remove` ?
It's not clear that we deal with examples.

I would rename theses routes like this:
`delete /api/examples/:exampleId': 'ExampleController.deleteExample`
`post /api/examples/:exampleId': 'ExampleController.updateExample`

No need for the `/remove` and `/update` the http method tells us the action to do